src/main/java/templates/mapPages.soy

```diff
@@ -0,0 +1,84 @@
+{namespace templates.pages}
+
+{template .mapPage}
+    {@param groupNames : list<string>}
+
+    <html>
+        <head>
+            <meta charset="UTF-8">
+            <meta name="viewport" content="width=device-width,initial-scale=1,shrink-to-fit=no">
+            <title>The Solidarity Initiative - Event Map</title>
+            <link rel="stylesheet" href="mapPage.css">
+            <link href="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.css" rel="stylesheet">
+            <script src="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.js"></script>
+            <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCm3MiL3acn2Y9ZoHXfCOWVvnJCzg2m3Us"></script>
+            <script src="mapScript.js"></script>
+            <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+        </head>
+
+        <body onload="initMap();">
+  
+            <aside class="mdc-drawer">
+                <div class="mdc-drawer__content">
+                    <nav class="mdc-list">
+                        <a class="mdc-list-item mdc-list-item--activated" href="#" aria-current="page">
+                            <i class="material-icons mdc-list-item__graphic" aria-hidden="true">map</i>
+                            <span class="mdc-list-item__text"><h2>Event Map</h2></span>
+                        </a>
+
+                        <hr class="mdc-list-divider">
+                        <h6 class="mdc-list-group__subheader">Relocate Map</h6>
+
+                        <a class="mdc-list-item" href="#">
+                            <label class="mdc-text-field mdc-text-field--filled">
+                                <span class="mdc-text-field__ripple"></span>
+                                <input class="mdc-text-field__input" id="address" type="text" aria-labelledby="my-label-id">
+                                <span class="mdc-line-ripple"></span>
+                            </label>  
+                        </a>
+
+                        <a class="mdc-list-item" href="#">
+                            <button class="mdc-button" id="submit">
+                                <div class="mdc-button__ripple"></div>
+                                <span class="mdc-button__label">Update Map</span>
+                                <i class="material-icons mdc-button__icon" aria-hidden="true">edit_location</i>
+                            </button>
+                        </a>
+
+                        <hr class="mdc-list-divider">
+                        <h6 class="mdc-list-group__subheader">Groups</h6>
+
+                        <div class="selection-controls-container">
+           
+                            {for $groupName in $groupNames}
+                            <div>
```

Comment:
What does this extra div here do? Also can you indent everything in for loop once so it's nested inside?

Reply:
The div makes the check marks vertical as opposed to horizontal. I've tried it without them and it made the groups in a row as opposed to a column.